openslides_votecollector/serializers.py

```diff
@@ -50,18 +50,17 @@ class Meta:
             'number',
             'seating_plan_x_axis',
             'seating_plan_y_axis',
-            'is_active',
+            'keypad_id',
         )
 
-    def get_is_active(self, obj):
+    # Returns the database id of corresponding keypad as read only field
+    def get_keypad_id(self, obj):
         keypad = Keypad.objects.filter(seat_id=obj.id)
-        if keypad and keypad[0].user:
-            is_active = keypad[0].user.is_present
-        elif keypad and not keypad[0].user:
-            is_active = True
+        if keypad and keypad[0]:
+            keypad_id = keypad[0].id
         else:
-            is_active = False
-        return is_active
+            keypad_id = None
+        return keypad_id
```

Comment:
Try `obj.keypad.id`. This should be the related name of Keypad from point of view of Seat.